assets/sass/elements/_elements-typography.scss

```diff
@@ -201,10 +201,12 @@ p,
   &:link,
   &:visited,
   &:hover,
+  a#{&}:focus,
```

Comment:
it evaluates the parent selector (.link-back) and applies outputs it after the a tag resulting in a.link-back:focus which neutralizes the a:link:focus set in govuk_template 
introduced in sass 3.3